shopinvader/services/abstract_sale.py

```diff
@@ -44,9 +44,14 @@ def _is_item(self, line):
 
     def _convert_one_line(self, line):
         variant = line.product_id._get_invader_variant(
-            self.shopinvader_backend,
-            self.env.context.get("lang", line.order_id.partner_id.lang),
+            self.shopinvader_backend, self.env.context.get("lang")
         )
+        if not variant:
+            # this likely should never happen if the request from client
```

Comment:
Maybe we should log something? (with info or warning)
Something like : Variant not found in the lang of the context : fr_FR use lang of partner en_US